week-1-practise/level-1/3-syntax-error-3.js

```diff
@@ -1,6 +1,6 @@
 // There is a syntax error with this code. Fix it.
 
-function(a, b, c) {
+function addNumbers(a, b, c) {
```

Comment:
Interestingly there is another way to solve this, search for Javascript and anonymous functions